### ISSUE TITLE
should code blocks be executed after pretty print?

### ISSUE THREAD

**Issue by no-stack-dub-sack** [Sketch-sh/sketch-sh#177]
sometimes in working in a code block, it's nice to pretty print your code, but if you're still working on something, code execution might not always be desirable. With code execution just another shortcut away, I think it might be better to leave that up to the user, and have pretty print be a standalone command with no execution. 

Thoughts?

**Comment by thangngoc89** [comment]
This is a really interesting point. I want to execute the code after pretty print to avoid layout thrashing. Also, you can't pretty print code with syntax error anyway.

**Comment by fstiffo** [comment]
I agree with @no-stack-dub-sack , and I would like it was possible to pp only one one block, for example the one with the active editing and having a button for the total pp.
